code/game/machinery/doors/airlock.dm

```diff
@@ -191,10 +190,8 @@
 		P.icon = icon
 		P.icon_state = part_id
 		P.name = name
-	if(mask_filter)
-		filters -= mask_filter
-	mask_filter = filter(type="alpha",icon=mask_file,x=mask_x,y=mask_y)
-	filters += mask_filter
+	remove_filter("mask_filter")
```

Comment:
on this note i am not sure for what exact reason the original if(mask_filter) was for as this proc is litterally only called in initial so there is no way for this to have anything in it before the add_filter but i replace it with remove_filter() that does exactly the same